### DISCUSSION TITLE
Not able to upload assets into Wiki.js which are larger then ~2.6 MB

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6554] [Help / Questions]
# Specs of my environment

- **No use of a reverse proxy**
- Current version of Wiki.js: 2.5.299
- Current version of Node.js: 16.20.0
- Current version of PostgreSQL: 11.16
- Operating System: Docker Container (Linux)

## Installation method

Using the official guide from [here](https://docs.requarks.io/install/ubuntu). Only exception is that the host OS is Debian 12 Bookworm.

## Changed settings

*Administration* > *Security* > *Max upload size*: `2622000`. But this setting is needless as I don't use something ~~then~~ like a reverse proxy.

The above change has no effect on the shown value within the Assets dialog which still states 5 MB

Used file types are mostly PNG files.

Above mentioned issue is present via inhouse and web access. The question is: How/where can I overcome this behavior?

**Comment by NGPixel** [comment]
The value shown in the dialog is static and won't reflect the actual value. You should still be able to upload larger files up to the limit you set.

**Reply by thamow** [reply]
Is there a way to change this in the future to reflect the real values from the database? The static text has caused a lot of confusion in the past and has led me (and seemingly other people) to take a look inside the db entry to ensure the settings are stored at all.
If that is not possible, maybe it would be a good idea to prefix or denote the text with "Max 10 files, 5 MB each **(by default if not changed)**"

**Comment by ghost** [comment]
@NGPixel thanks for the hint regarding the *max filesize value*. Do you have any idea why I am limited in uploading assets as mentioned above?

Is there a setting I need to change to solve the mentioned problem. I have already searched the official documentation as well as open/closed issues in the Github repo with no luck.

**Reply by thamow** [reply]
Take a look at this comment. Does this help you?
Additionally, if that was not clear to you, regardless of whether you are using a reverse proxy or not, you still need to change the setting for max upload size.

https://github.com/requarks/wiki/discussions/7077#discussioncomment-9997807
